charts/opentelemetry-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.114.0
+    tag: 0.115.0
```

Comment:
i think we need to use the [115.1 image](https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.115.1) here